azure-pipelines.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+
+err=[]
+
+if os.environ.get('testWithPip') == 'true':
+    err.append(subprocess.run("python setup.py test", shell=True).returncode)
```

Comment:
I think this whole file gets a bit less verbose if we move ``err.append(subprocess.run(CMD).retcode)`` to a separate function taking as input the parameter CMD

Reply:
Thanks! This is the kind of thing that I as a sysadmin rather than programmer tend to miss :-)

Implemented, I think the correct way but please educate me if I'm writing bizarre python...